Main layout html form

Nyaroslav opened the pull request:
https://Nyaroslav.github.io/layout_html-form - [DEMO LINK]
[https://Nyaroslav.github.io/layout_html-form/report/html_report/](url) - [TEST REPORT LINK]

Denchepornyuk commented on src/index.html:
```diff
@@ -1,15 +1,189 @@
+  </head>
+
+  <body>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="POST">
```
для атрибутів потрібно добавляти відступи, щоб вони не були на одному рівні з тегом

Denchepornyuk commented on src/index.html:
```diff
@@ -1,15 +1,189 @@
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
```
```suggestion
    <meta
      name="viewport"
      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
    >
```

Denchepornyuk commented on src/index.html:
```diff
@@ -1,15 +1,189 @@
+              type="date"
+              name="birth"
+              id="birth"
+              >
```
```suggestion
            >
```

Denchepornyuk commented on src/index.html:
```diff
@@ -1,15 +1,189 @@
+              value="12"
+              min="1"
+              max="100"
+              >
```
```suggestion
            >
```

Denchepornyuk commented on src/index.html:
```diff
@@ -1,15 +1,189 @@
+              name="surname"
+              id="surname"
+              autocomplete="off"
+              >
```
закриваюча скобка має бути на рівні з тегом

Denchepornyuk replied:
```suggestion
            >
```

Denchepornyuk commented on src/index.html:
```diff
@@ -1,15 +1,189 @@
+        </div>
+      </fieldset>
+    </form>
+    <button
+    type="submit"
+    onsubmit="onSubmit()"
+    class="block"
+    >
```
а де відступи для атрибутів ?

Denchepornyuk commented on src/index.html:
```diff
@@ -1,15 +1,189 @@
+    onsubmit="onSubmit()"
+    class="block"
+    >
+    Submit
```
для текста також добавляй відступ, щоб він не був на одному рівні з тегом

Nyaroslav committed:
added layout

Nyaroslav committed:
fixed code structure

Nyaroslav committed:
demo link

Nyaroslav committed:
div to label and [DEMO LINK](https://Nyaroslav.github.io/layout_html-form/)

Nyaroslav committed:
- [DEMO LINK](https://Nyaroslav.github.io/layout_html-form/)
- [TEST REPORT LINK](https://Nyaroslav.github.io/layout_html-form/report/html_report/)

slavanosatskyi commented on src/index.html:
```diff
@@ -1,17 +1,209 @@
+          <label 
+            class="label" 
+            for="radioButton1"
+          >Yes</label>
```
please, check code style for other labels as well

```suggestion
          <label 
            class="label" 
            for="radioButton1"
          >
            Yes
         </label>
```

slavanosatskyi reviewed (changes requested):
Keep it up, almost done

this requirement doesn't work for all inputs (probably you just forget to redeploy your solution, as i don't see labels elements in you demo, just text)
![Screenshot 2023-03-02 at 17 45 08](https://user-images.githubusercontent.com/6526149/222477469-9aaad7ca-dad8-42cc-9697-d574a6e9ffcd.png)
this doesn't work as well - On form submit send form data to https://mate-academy-form-lesson.herokuapp.com/create-application

and please, provide valid report link

Nyaroslav committed:
fix

Nyaroslav committed:
fix2
